src/store/navMenuTodo.ts

```diff
@@ -0,0 +1,59 @@
+import { AnyAction } from 'redux';
+import { NavType } from '../api/interface';
+
+export const ALL_TODOS = 'All_TODOS';
+export const COMPLETED_TODOS = 'COMPLETED_TODOS';
+export const ACTIVE_TODOS = 'ACTIVE_TODOS';
+export const FILTERING = 'FILTERING';
+
+export function allTodosAction() {
+  return {
+    type: ALL_TODOS,
+  };
+}
+
+export function completedTodosAction() {
+  return {
+    type: COMPLETED_TODOS,
+  };
+}
+
+export function activeTodosAction() {
+  return {
+    type: ACTIVE_TODOS,
+  };
+}
+
+export function filteringTodosAction(query: string) {
+  return {
+    type: FILTERING,
+    query,
+  };
+}
+
+const initialState: NavType = {
+  filteringType: ALL_TODOS,
+  query: '',
+};
+
+export function navReducer(state: NavType = initialState, action: AnyAction) {
```

Comment:
```suggestion
export function todosReducer(state: NavType = initialState, action: AnyAction) {
```
почему navReducer? Это же не навигация

Reply:
А это переместить в todos reducer